README.md

```diff
@@ -2,6 +2,37 @@
 
 This is an easy way to develop with the agile stack on real hardware.
 
+## Modifications for this branch
+
+###  This branch contains the following modified components
+
+* OS-JS:
+ * integrate with login interface 
+ * propagates the token to node-red through the node-red frame application
+* Node-RED 
+ * run an express app
+ * get the token from agile-idm in the initial request to the application and verify that only a particular user from IDM can use a node-red instance by mapping the user session to  the admin session from Node-RED
+ * this node-red includes the agile-idm token node to retrieve the token of the currently logged in user.
+* AGILE-IDM
+
+### How to configure it
+
+You need to add something like this to your /etc/hosts file (assuming that the gateway is on 192.168.1.20):
+
+192.168.1.20   agilegw.local
+
+This is needed because the configurations of all the modified components rely in this name (agilegw to find each other; especially, when this comes to Oauth2 integrations). We should address this in the future in the project (see https://github.com/Agile-IoT/Architecture/issues/3)
```

Comment:
True. The issue was [https://github.com/Agile-IoT/Architecture/issues/4]. But, this part is also removed from the README now.